chargeio/services/RecurringCharges/RecurringChargeService.cs

```diff
@@ -1,62 +1,53 @@
 ﻿using System;
+using ChargeIo.services;
 using ChargeIO.Infrastructure;
 using ChargeIO.Models;
 
 namespace ChargeIO.Services.RecurringCharges
 {
-	public class RecurringChargeService
-	{
-        private string SecretKey { get; }
-        
-        public RecurringChargeService(string secretKey = "")
+    public sealed class RecurringChargeService : ServiceBase

```

Comment:
We shouldn't be sealing our classes -- we shouldn't be restricting users from inheriting from our classes.